pycasbin/README.md

```diff
@@ -0,0 +1,129 @@
+## Language Bindings for Casbin
+
+Casbin-CPP provides language bindings to compound the advantages of different languages and to make 
+authorization easier and faster.
+
+At present, casbin-cpp provides language bindings for Python.
+
+## Python Bindings
+
+### Installing the PyCasbin module
+
+It is assumed you have CMake >=v3.19 and Python >= 3.2 installed.
+
+1. Clone/download the project:
+    ```bash
+    git clone https://github.com/casbin/casbin-cpp.git
+    ```
+
+2. Make a build directory and generate project files through CMake:
+    ```bash
+    mkdir build
+    cd build
+    cmake ..
+    ```
+    **Note:** Kindly look at the log message to find the directory you need to add in your `sys.path` (Step 5). The log may look like this:
+    ```bash
+    [pycasbin]: Build "pycasbin" target for Python Bindings
+    [pycasbin]: Add "lib/python3.9/site-packages" to your sys.path/USER_SITE variable if not already present
+    ```
+
+3. Build the python bindings (`pycasbin` target):
+    ```bash
+    cmake --build . --config Release --target pycasbin
+    ```
+
+4. Install the `pycasbin` module:
```

Comment:
We also need to update the docs for installing through pip. Should we do it in this PR itself? @hsluoyz

Reply:
Sure @sheny1xuan